src/main/java/eu/tornplayground/tornapi/models/faction/basic/Rank.java

```diff
@@ -0,0 +1,59 @@
+package eu.tornplayground.tornapi.models.faction.basic;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import eu.tornplayground.tornapi.models.Model;
+
+import java.util.Objects;
+
+public class Rank extends Model {
+    @JsonProperty("level")
+    private int level;
+
+    @JsonProperty("name")
+    private String name;
```

Comment:
Limited value, could be a string.

[Unofficial docs](https://tornapi.tornplayground.eu/faction/attacks#attack_result)

Reply:
Not sure what you mean... 🤔

Reply:
Will help when I use the correct words. Could be an enum. Really shouldn't be reviewing more code after such a chaotic day and week at work ....

Reply:
Ahhhh yes i see.. ^^

Reply:
made it an enum